src/VerseSuggesting.ts

```diff
@@ -39,6 +39,9 @@ export class VerseSuggesting implements IVerseSuggesting {
   public get ReplacementContent(): string {
     let head = `> [!Bible]`
     let bottom = ''
+    if (this.settings?.minimalOutput) {
+      head = ''
+    }   
     if (this.settings?.collapsibleVerses) {
```

Comment:
not sure, will these 2 settings conflicts?

like when minimalOutput and collapsibleVerses both be true?


Would you mind sharing some screeshots?

Reply:
@ajrty33 

Thanks for your screenshot.

To clarify, my concern is what about both of these settings are true.

What the UI will be like?

It looks like the second condition will override the first one, which is your change.

Reply:
Can you test it that for me?

Or just share a screenshot when both of they are true.